db/migrate/20200514052159_add_status_to_allocations.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToAllocations < ActiveRecord::Migration[5.2]
+  def change
+    add_column :allocations, :status, :string
```

Comment:
probably should have `null: false, index: true`

Reply:
we are allowing `nil` allocations for now as we only set the cancelled flag, we need to iron out the other statuses post MVP

Reply:
ok for null status for MVP 👍